core/themes/basis/templates/comment.tpl.php

```diff
@@ -67,13 +70,18 @@
   <div class="comment-text">
 
     <div class="comment-title">
-      <?php print render($title_prefix); ?>
-      <h3><?php print $title; ?></h3>
-      <?php print render($title_suffix); ?>
+      <?php if ($title_display == TRUE): ?>
```

Comment:
This could be shortened (doesnt have to be so, just saying) to 
`<?php if ($title_display): ?>`